.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release maven package
+on:
+  workflow_dispatch:
+
+env:
+    SIGN_KEY: ${{ secrets.SIGN_KEY }}
+    SIGN_KEY_ID: ${{ secrets.SIGN_KEY_ID }}
+    SIGN_KEY_PASS: ${{ secrets.SIGN_KEY_PASS }}
+
+jobs:
+  release_maven_package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate branch
+        run: |
+          echo "Release can only be triggered from master branch"
+          exit 1
+          if: ${{ github.ref != 'refs/heads/master' }}
+
+      - uses: actions/checkout@v3
+
+      - name: Set up jdk
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '8'
+
+      - name: Maven prepare release
+        run: |
+          mvn \
+          -P release \
+          -DsonatypeOssDistMgmtSnapshotsUrl=${{ secrets.SONATYPE_OSS_DIST_MGMT_SNAPSHOTS_URL }} \
+          -B clean release:prepare 
```

Comment:
This would try to run an interactive release process to gather the version numbers, no? I think we need to pass at least the `releaseVersion` and `developmentVersion` properties to make the process non-interactive. (see: https://maven.apache.org/maven-release/maven-release-plugin/examples/non-interactive-release.html)

We can use inputs for the `workflow_dispatch` event to ask for the values.

Reply:
Ah okay, I wasn't aware of that. So it automatically increases the patch version? (`0.9.61`, `0.9.62`, `0.9.63`, ...)

Since this library is in maintenance mode, we will probably not bump the major or minor version anytime soon. So we can probably leave it as it is. Should we have to bump the major or minor, we need to adjust the workflow.

Reply:
> Should we have to bump the major or minor, we need to adjust the workflow.

You can just manually push a pom with the new version x.x-SNAPSHOT manually before the release:prepare without adjusting the workflow.

Reply:
Ah, nice. Thanks! :+1: